dev_notes/Penetration Test Assesment.md

```diff
@@ -0,0 +1,90 @@
+## Risk Identification
+1. Identify assets (e.g. web application)
+    1. Web application
+    2. Api
+    3. Docker Hub
+    4. Github repository 
+    5. Digital Ocean
+    6. Grafana / Prometheus / Kibana
```

Comment:
those are only service exposed through ports? so Elasticsearch shouldn't be here?